scripts/ansible/roles/linux/openenclave/tasks/environment-setup-cross-arm.yml

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
```

Comment:
Just a really small nit, all the files have this header:
```
# Copyright (c) Microsoft Corporation. All rights reserved.
# Licensed under the MIT License.

---
```
and this file lacks the line:
```
---
```